packages/url_launcher/url_launcher_android/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncher.java

```diff
@@ -46,12 +48,12 @@ interface IntentResolver {
   }
 
   UrlLauncher(@NonNull Context context) {
-    this(
-        context,
+    this.applicationContext = context;
+    this.intentResolver =
```

Comment:
Why are you changing this to remove the constructor delegation?

Reply:
reverted